src/main/java/by/byshnev/services/ArticleServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package by.byshnev.services;
+
+import by.byshnev.dao.ArticleDAOImpl;
+import by.byshnev.dto.ArticleDto;
+import by.byshnev.mappers.ArticleMapper;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@RequiredArgsConstructor
+@Service
+public class ArticleServiceImpl implements ArticleService {
+
+    private final ArticleMapper articleMapper;
+
+    private final ArticleDAOImpl articleDAO;
```

Comment:
always inject **interfaces** instead of particular classes. service doesn't care what actual dao it uses.